macros/PGbasicmacros.pl

```diff
@@ -1410,19 +1410,13 @@ =head2 Display Macros
               Latex2HTML => "output this in Latex2HTML mode",
              )
 
-    TEX     (tex_version, html_version) #obsolete
-
     M3      (tex_version, latex2html_version, html_version) #obsolete
 
 =cut
 
-sub TEX {
-	my ($tex, $html ) = @_;
-	MODES(TeX => $tex, HTML => $html, HTML_tth => $html, HTML_dpng => $html);
-}
 
 sub M3 {
-	my($tex, $l2h, $html) = @_;
+	my($tex,$l2h,$html) = @_;
```

Comment:
In https://github.com/rederly/pg/pull/1/files you commented out the entire `M3` function.
If it is really obsolete and can be removed - maybe remove it here also.
If removing it needs to be deferred - maybe we should try to at least issue a deprecation warning, so it can be removed in the future.

Reply:
The description for M3 says it is deprecated, and perhaps you're right - a warning message is appropriate here?

Reply:
Leaving as is. Marking resolved.